### ISSUE TITLE
Great engine, could be extended with character

### ISSUE THREAD

**Issue by bluebabbler** [unruhe73/python-adventure-game-engine#1]
Great game engine @unruhe73 ! I think it could be extended with characters that can talk and interact with the player. It is possible to describe them someway with objects but it is rather clumsy (and I don't like to have characters described as objects).

**Comment by unruhe73** [comment]
Yes. It has to be extended surely with characters and dialogues. It's in my TODOs list. There are a lot of features that can be implemented.
